### DISCUSSION TITLE
Add potion timer reset upon death

### DISCUSSION THREAD

**Discussion by ander210** [runelite/runelite#15170] [Ideas]
I find the potion timer extremely useful and easy to keep track with. Upon death, the Runelite plugin continues with the previous timer, but when you die in game, the stats restore. It is just a suggestion to reset the timers aswell so it would be a more clear visual sign.

**Comment by pwatts6060** [comment]
I thought this at first as well. However which potions is this for? Divine, antifire, antipoison actually retain there effects even after death. you can boost with regular super pots to get back the divine boost, e.g.

**Reply by ander210** [reply]
I wrote this after my most recent death (which was just now). I noticed it happen with divine potions. Super combat and ranging both reset. I looked up from my stats and they have gone down to what my levels actually are.

If the effects are retained then its rather a question of why the numbers go down.
